prov/verbs/src/verbs_eq.c

```diff
@@ -201,6 +180,17 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq, struct rdma_cm_event *cma_event
 	case RDMA_CM_EVENT_REJECTED:
 		eq->err.err = ECONNREFUSED;
 		eq->err.prov_errno = -cma_event->status;
+		eq->err.err_data = NULL;
+		eq->err.err_data_size = 0;
+		if (cma_event->param.conn.private_data_len) {
+			cm_hdr = cma_event->param.conn.private_data;
+			eq->err.err_data = calloc(1, cm_hdr->size);
```

Comment:
Can you define a structure that's large enough to hold the event data, so we can avoid these extra memory allocations?